MinnetonkaGame.py

```diff
@@ -41,12 +43,44 @@ def big_room():
         game_over("They ask you to show your skill by timing you on a Minecraft speed run. " +
                   "You fail miserably as you have never played before.")
     elif answer == "2":
-        # TODO: top_of_stairs_landing()
+        top_of_stairs_landing()
         game_over("TBD")
```

Comment:
This game_over() line can be removed as we now branch off to top_of_stairs_landing().

Reply:
removed and commited